common/src/main/resources/suithemes/default/elements/button.variables

```diff
@@ -256,28 +256,28 @@
 @tertiaryLinePadding: 0.5em;
 @tertiaryLineHeight: 0.2em;
 @tertiaryTextColor: @textColor;
-@tertiaryLineColor: lighten(@tertiaryTextColor, 20%);
+// @tertiaryLineColor: lighten(@tertiaryTextColor, 20%);
```

Comment:
Shall we remove the commented lines?

Reply:
Actually no, I have to comment these out to get css variables to work, but eventually when there is a fomantic update we'd need to replace the default theme and I want to track what was commented